server.js

```diff
@@ -30,6 +30,7 @@ var queueNumber = 1;
 const vuileFritsTime = 9000;
 const vuileFritsTimeout = 2000;
 const baudetTime = 10000;
+const aasKoningTimeout = 4000;
 const disconnectTimeout = 20000;
```

Comment:
maybe make this into an object

const DURATIONS= {
  vuileFrits: 9000,
  baudet: 10000
}

const TIMEOUTS= {
  vuileFrits: 9000,
  aasKoning: 4000,
  disconnect: 20000
}